test/evergreen.yml

```diff
@@ -2844,7 +2844,7 @@ buildvariants:
   tasks:
     - name: compile
     - name: make-check-test
-    - name: unit-test-with-compile
```

Comment:
There's a reason we use `unit-test-with-compile` task (instead of `unit-test`) for macOS builder - details see WT-7239.

Reply:
I changed it to `unit-test` since with the latest commits the `compile` and `unit-test` steps are running on the same distro. This ticket you linked seems to be related to mismatched toolchains introduced by running `compile` and `unit-test` across different macos distros?

Reply:
You are right that the issue in WT-7239 is not applicable since the `compile` and `unit-test` tasks are now scheduled on the same macOS distro. All good then!